owners/src/parser.js

```diff
@@ -90,9 +90,12 @@ class OwnersParser {
     if (owner.startsWith('?')) {
       modifier = OWNER_MODIFIER.SILENT;
       owner = owner.slice(1);
-    } else if (owner.startsWith('!')) {
+    } else if (owner.startsWith('#')) {
       modifier = OWNER_MODIFIER.NOTIFY;
       owner = owner.slice(1);
+    } else if (owner.startsWith('!')) {
+      modifier = OWNER_MODIFIER.REQUIRE;
```

Comment:
I assume the code that processes this modifier is in a different PR?

Reply:
Yes; this is the parser-only side. I was working on them together in the same branch, but decided to split out this part since it's self-contained.